index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <link rel="stylesseet" href="styles.css" >
+  </head>
+
+  <body>
+<section id ="welcome-section">
+  <h1> Texto </h1>
+</section>
+<section id ="projects">
+  <h2 class="project-tile"> Projects</h2>
+  <a href="www.google.com"> link</a>
+  </section>
+  <nav id="navbar">
+    <a href="#welcome-section"> link dentro de la pagina</a>
+  </nav>
+  <a id="profile-link" href="https://github.com/montsecruz9" target="_blank"> Link a mi perfil </a>
+  </body>
+  
+
+</html>
```

Comment:
☺️ I kindly recommend you to use the indentation, this will help your code to be clean and readable, you will also be promoting the use of good practices.
```suggestion
<body>
  <section id ="welcome-section">
    <h1> Texto </h1>
  </section>
  <section id ="projects">
    <h2 class="project-tile"> Projects</h2>
    <a href="www.google.com"> link</a>
  </section>
  <nav id="navbar">
    <a href="#welcome-section"> link dentro de la pagina</a>
  </nav>
    <a id="profile-link" href="https://github.com/montsecruz9" target="_blank"> Link a mi perfil </a>
    </body>
  
  </html>
```